js/src/components/faqs-panel/index.js

```diff
@@ -30,11 +31,12 @@ const getPanelToggleHandler = ( trackName, id ) => ( isOpened ) => {
  * @param {Object} props React props.
  * @param {string} props.trackName The track event name to be recorded when toggling on FAQ items.
  * @param {Array<FaqItem>} props.faqItems FAQ items for rendering.
+ * @param {string} props.className The class name for this component.
```

Comment:
💅  

props.className  Is optional

Reply:
Thanks, updated in 43a1d02.

Reply:
I still see reduced space in 

`How do I sync my products to Google free listings?` and `Can I run both Shopping ads and free listings campaigns at the same time?`

![Uploading Screenshot 2022-05-04 at 11.56.06.png…]()